third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -1662,28 +1662,6 @@ function previewBlinkStyle(style) {
   };
 }
 
-function previewBlinkRule(rule) {
```

Comment:
This looked unused, and it triggered `no-undef` with the `getObjectIdRaw` call on line 1673, so I removed the entire function.

Reply:
Hum...

* This should be covered by `convertCdpToRrpCssRules`.
* From the looks of it, this is a modified copy-and-paste from Gecko, since it refers to `InspectorUtils` (see [here](https://github.com/replayio/gecko-dev/blob/628cc55f22785f3a66a8c767cdc86f31feb9a050/layout/inspector/InspectorUtils.cpp#L155)).